frog/imports/ui/Preview/Controls.jsx

```diff
@@ -18,26 +16,6 @@ import { initActivityDocuments } from './Content';
 import { activityTypesObj } from '../../activityTypes';
 import { addDefaultExample } from './index';
 
-const styles = () => ({
-  root: {
-    padding: '4px',
-    margin: '4px'
-  },
-  closeButton: {
-    float: 'right',
-    marginRight: '5px',
-    backgroundColor: 'red',
-    color: 'white'
-  },
-  exampleButton: {
-    textTransform: 'none'
-  },
-  exampleButtonDeeplink: {
-    textTransform: 'none',
-    fontStyle: 'italic'
-  }
-});
```

Comment:
We have decided to use css through material-ui's `withStyle` for the whole project. We gradually remove all of the inline css to be replaced by withStyle. The advantages I see are that it is easier to work with a single way, over the whole project instead of having several ways of doing it, also using `withStyles` makes the component easier to read. I prefer to see "This component has the css called `closeButton`" rather than "This component has the css `{ float: 'right', marginRight: '5px', backgroundColor: 'red', color: 'white' }`" when I read the code of a component.

Reply:
Makes sense to me. Thanks !